test/repo-sync.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash -xe
+# Copyright 2016 The Kubernetes Authors All rights reserved.
```

Comment:
Not sure if the boilerplate is wrong because the year is wrong, or maybe a space between lines 1&2 is necessary?